tests/link/pytorch/test_basic.py

```diff
@@ -343,3 +380,44 @@ def test_pytorch_OpFromGraph():
 
     f = FunctionGraph([x, y, z], [out])
     compare_pytorch_and_py(f, [xv, yv, zv])
+
+
+def test_ScalarLoop_Elemwise():
```

Comment:
Since there's a special condition for one or multiple carry, please test also both kinds of loop with multiple updates

Reply:
Done.